[2022-08-29T19:26:07.367-0400] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: dag_with_catchup_backfill_v02.task1 backfill__2022-08-12T00:00:00+00:00 [queued]>
[2022-08-29T19:26:07.382-0400] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: dag_with_catchup_backfill_v02.task1 backfill__2022-08-12T00:00:00+00:00 [queued]>
[2022-08-29T19:26:07.383-0400] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-08-29T19:26:07.383-0400] {taskinstance.py:1369} INFO - Starting attempt 1 of 6
[2022-08-29T19:26:07.383-0400] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-08-29T19:26:07.424-0400] {taskinstance.py:1389} INFO - Executing <Task(BashOperator): task1> on 2022-08-12 00:00:00+00:00
[2022-08-29T19:26:07.427-0400] {standard_task_runner.py:52} INFO - Started process 10129 to run task
[2022-08-29T19:26:07.431-0400] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_catchup_backfill_v02', 'task1', 'backfill__2022-08-12T00:00:00+00:00', '--job-id', '31', '--pool', 'default_pool', '--raw', '--subdir', '/home/augenz/PycharmProjects/airflow/dags/dag_with_catchup_and_backfill_v02.py', '--cfg-path', '/run/user/1000/app/com.jetbrains.PyCharm-Community/tmp5enezc43', '--error-file', '/run/user/1000/app/com.jetbrains.PyCharm-Community/tmpzggz0iee']
[2022-08-29T19:26:07.433-0400] {standard_task_runner.py:80} INFO - Job 31: Subtask task1
[2022-08-29T19:26:07.656-0400] {task_command.py:371} INFO - Running <TaskInstance: dag_with_catchup_backfill_v02.task1 backfill__2022-08-12T00:00:00+00:00 [running]> on host pop-os.localdomain
[2022-08-29T19:26:07.935-0400] {taskinstance.py:1581} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=dag_with_catchup_backfill_v02
AIRFLOW_CTX_TASK_ID=task1
AIRFLOW_CTX_EXECUTION_DATE=2022-08-12T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=backfill__2022-08-12T00:00:00+00:00
[2022-08-29T19:26:07.937-0400] {subprocess.py:62} INFO - Tmp dir root location: 
 /run/user/1000/app/com.jetbrains.PyCharm-Community
[2022-08-29T19:26:07.938-0400] {subprocess.py:74} INFO - Running command: ['/usr/bin/bash', '-c', 'echo This is a simple bash command!']
[2022-08-29T19:26:07.948-0400] {subprocess.py:85} INFO - Output:
[2022-08-29T19:26:07.952-0400] {subprocess.py:92} INFO - This is a simple bash command!
[2022-08-29T19:26:07.952-0400] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-29T19:26:08.061-0400] {taskinstance.py:1407} INFO - Marking task as SUCCESS. dag_id=dag_with_catchup_backfill_v02, task_id=task1, execution_date=20220812T000000, start_date=20220829T232607, end_date=20220829T232608
[2022-08-29T19:26:08.126-0400] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-29T19:26:08.273-0400] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-29T19:26:08.350-0400] {dagrun.py:567} INFO - Marking run <DagRun dag_with_catchup_backfill_v02 @ 2022-08-12 00:00:00+00:00: backfill__2022-08-12T00:00:00+00:00, state:running, queued_at: None. externally triggered: False> successful
[2022-08-29T19:26:08.351-0400] {dagrun.py:612} INFO - DagRun Finished: dag_id=dag_with_catchup_backfill_v02, execution_date=2022-08-12 00:00:00+00:00, run_id=backfill__2022-08-12T00:00:00+00:00, run_start_date=2022-08-29 23:25:29.710622+00:00, run_end_date=2022-08-29 23:26:08.351412+00:00, run_duration=38.64079, state=success, external_trigger=False, run_type=backfill, data_interval_start=2022-08-12 00:00:00+00:00, data_interval_end=2022-08-13 00:00:00+00:00, dag_hash=None
